person.rb

```diff
@@ -1,20 +1,27 @@
+require './corrector'
+
 class Person
   attr_accessor :name, :age, :permission
   attr_reader :id
 
-  def initialize(age, name = 'Unknown', permission: true)
+  def initialize(age, name, permission: true)
     @id = Random.rand(1..1000)
     @name = name
+    @corrector = Corrector.new(name)
```

Comment:
- [ ] When initiating the `Corrector` class here you are sending the `name` as an argument instead of the instance variable `@name`.

Reply:
Thanks @mikethreels  for the feedback 👍